src/main/resources/application.properties

```diff
@@ -162,3 +163,7 @@ ccd.am.read.from_am=${CCD_AM_READ_FROM_AM:}
 # http statuses with comma separated to ignore audit logs
 audit.log.ignore.statues=404
 audit.log.enabled=true
+
+# Role logging
+role.logging.enabled=${DATA_STORE_ROLE_LOGGING_ENABLED:false}
```

Comment:
I was thinking to have only a log-level alignment, rather than a logical switch to enable/disable the related logging at all.